Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -395,7 +399,7 @@ define([
 
         var cullingVolume = frameState.cullingVolume;
 
-        var boundingVolume = surfaceTile.boundingSphere3D;
+        var boundingVolume = defaultValue(surfaceTile.boundingOBB, surfaceTile.boundingSphere3D);
 
         if (frameState.mode !== SceneMode.SCENE3D) {
```

Comment:
Yes, but we can do it later.  Let's test the performance improvement first and then see how involved this is.  It could be easy, but I am not familiar with all the code.